packages/neos-ui/src/manifest.js

```diff
@@ -219,6 +219,13 @@ manifest('main', {}, globalRegistry => {
         store.dispatch(actions.CR.Workspaces.update(feedbackPayload));
     });
 
+    //
+    // When the server advices to update the nodes preview URL, dispatch the action to do so
+    //
+    serverFeedbackHandlers.set('Neos.Neos.Ui:UpdateNodePreviewUrl/Main', (feedbackPayload, {store}) => {
+        store.dispatch(actions.UI.ContentCanvas.setPreviewUrl(feedbackPayload.newPreviewUrl));
```

Comment:
I think we still need to discuss thus part :)

Reply:
I think that is incorrect as soon as you publish multiple document nodes at once...

Reply:
very good catch indeed..